cpp/CMakeLists.txt

```diff
@@ -689,6 +688,39 @@ following IMPORTED GLOBAL  targets:
     ]=])
 
 
+set(common_code_string
+    [=[
+if(NOT TARGET cudf::Thrust)
+  thrust_create_target(cudf::Thrust FROM_OPTIONS)
+endif()
+
+# nvcc automatically adds the CUDA Toolkit system include paths before any
+# system include paths that CMake adds.
+#
+# CMake implicitly treats all includes on import targets as 'SYSTEM' includes.
+#
+# To get the cudacxx shipped with cudf to be picked up by consumers instead of the
+# version shipped with the CUDA Toolkit we need to make sure it is a non-SYSTEM
+# include on the CMake side.
+#
+# Todo this currently we move the includes from the cudf::cudf target to a
```

Comment:
Is this a TODO? Or are you saying "To do this currently, we move..."

Reply:
I am saying "To do this currently, ... ". I will revise the comment to be clear.